docs/manual/guieffect-checker.tex

```diff
@@ -66,7 +66,7 @@ \chapter{GUI Effect Checker\label{guieffect-checker}}
 @SafeEffect
 public void calledFromBackgroundThreads(JLabel l) \{
     l.setText("Foo");       // Error: calling a @UIEffect method from a @SafeEffect method
-    Display.syncExec(new @UI Runnable \{
+    Display.syncExec(new Runnable \{
```

Comment:
There is actually not a lot about `@UI` as a type use annotation. Unless you mean some sort of more general Checker Framework instruction elsewhere in the docs? I just had to change this example, you still need `@UI` if storing the class somewhere as described below.